docs/models/components/bodytype.md

```
@@ -0,0 +1,17 @@
+
+```typescript
+import { BodyType } from "@starmorph/sundial-exchange-api-typescript/models/components";
+
+let value: BodyType = "text";
+```
+
+## Values
+
+```typescript
+"json" | "form-data" | "multipart-form-data" | "text" | "binary"
+```
```

markdownlint-cli2 (0.18.1)

Files should end with a single newline character

(MD047, single-trailing-newline)